repository/src/main/resources/db/useful-queries/Remove all definitions for a case type older than 3 months, but leaving max version.sql

```diff
@@ -91,8 +91,8 @@ BEGIN
   DELETE FROM display_group WHERE case_type_id IN
     (SELECT id FROM tmp_case_type_ids);
 
-  DELETE FROM noc_config WHERE case_type_id IN
-      (SELECT id FROM tmp_case_type_ids);
+  --DELETE FROM noc_config WHERE case_type_id IN
```

Comment:
@dinesh1patel - you can delete these lines instead of commenting it out

Reply:
Done